litgpt/utils.py

```diff
@@ -403,6 +403,21 @@ def save_hyperparameters(function: callable, checkpoint_dir: Path) -> None:
     """Captures the CLI parameters passed to `function` without running `function` and saves them to the checkpoint."""
     from jsonargparse import capture_parser
 
+    # TODO: Make this more robust
```

Comment:
How do you think this could be done? Do we need to choose between `jsonargparse.CLI` or the CLI in `__main__` and then pass the correct one to `capture_parser`?

We could also simplify this by not having a CLI in the scripts themselves.

Would you open an issue to track this?